api/src/i18n/services/translation.service.spec.ts

```diff
@@ -58,12 +67,16 @@ describe('TranslationService', () => {
         {
           provide: SettingService,
           useValue: {
-            getSettings: jest.fn().mockResolvedValue({
-              chatbot_settings: {
-                global_fallback: true,
-                fallback_message: ['Global fallback message'],
+            getSettings: jest
+              .fn()
+              .mockResolvedValue(['Global fallback message']),
+            find: jest.fn().mockResolvedValue([
+              {
+                translatable: true,
+                key: 'global_fallback',
+                value: 'Global fallback message',
               },
-            }),
+            ]),
```

Comment:
Is this needed ? 

+ in settings we don't have "key"

Reply:
I think it's needed since it's being used in the refresh endpoint. When I remove it, the test 'should return an empty array from the settings when global fallback is disabled' doesn't pass